src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { TodoItem } from '../TodoItem';
+import { Todo } from '../../types/Todo';
+
+type Props = {
```

Comment:
```suggestion
interface Props  {
```
use interface prefer